### PR TITLE
Enabling FTUE splash carousel

### LINKED ISSUE #4584: Build FTUE Splash screens
### Your use case

When the app opens, and the user is not logged in, we show a Splash screen. 
- [_link to Figma splash screens_](https://www.figma.com/file/xdV4PuI3DlzA1EiBvbrggz/Login-Flow-v2?node-id=1628%3A31003)

**The Splash screens include:**
- ~A loading page~ Descoped 
- A landing page carousel that automatically rotates 4 images + text combinations every 5 seconds. The user can also swipe between screens. Once a user has manually swiped, the automatic transitions are disabled.
- 2 buttons, Get Started (create new account) & I already have an account (login)
  - While we are still building the FTUE experience Get Started will lead to … and I already have an account will lead to...

--- 
**Tasks**
- [x] Jordan to review the visuals used on screen
- [ ] Marketing to approve the copy used on screens
- Build
  - [x] I already have an account #4668 
  - [x] Carousel with 4 images, text combinations, I already have an account flow #4727 
  - [x] Automatic carousel rotation #4740
  - [x] Disable automatic carousel rotation after user interaction #4914 
  - [x] Update with final images/copy/dark mode gradients, styled title full stops #4880 
  - [x] Locking phones to portrait during FTUE auth #4918 
  - [ ] Enable feature #4964 
- [x] Design review before merge

---
_This issue is considered complete when the tasks are complete, the screen matches the Figma design, it's been reviewed (by Product, Design & Engineering), and merged._

### PR DESCRIPTION
**Pull request by ouchadam**
Fixes #4584 

Enabling the FTUE splash carousel flag, will also include any product/design review changes in this PR as well 

- Updates dark mode background to use a solid colour
- Replaces a `GET STARTED` with `CREATE ACCOUNT`  when the `I already have an account` feature is enabled

| 1 | 2 | 3 | 4 |
| --- | --- | --- | --- |
Screenshot_20220119_094008|Screenshot_20220119_094015|Screenshot_20220119_094022|Screenshot_20220119_094028
Screenshot_20220119_094041|Screenshot_20220119_094046|Screenshot_20220119_094057|Screenshot_20220119_094103

| TINY DEVICE | N9 TABLET LANDSCAPE | N9 TABLET PORTRAIT |
| --- | --- | --- |
small|Screenshot_20220119_094344|Screenshot_20220119_094333

_I already have an account_ 
| ENABLED | DISABLED |
| --- | --- |
|Screenshot_20220119_094008|Screenshot_20220119_094713

### CODE REVIEW

**Review by daniellekirkwood — APPROVED** [review]

**Review by bmarty — APPROVED** [review]
LGTM, does it require to update the allScreensTest() test?

**Comment by ouchadam** [comment]
@bmarty good point, will double check :+1:

**Comment by amshakal** [comment]
Looking good! :D The only changes that I'd request to be made are:
1. Change the 'm' in secure messaging to lowercase
2. Change 'GET STARTED' to 'CREATE ACCOUNT'

Thank you! 👯

**Review by amshakal — APPROVED** [review]

**Comment by ouchadam** [comment]
@bmarty there's been a small change to the PR to swap `GET STARTED` with `CREATE ACCOUNT` 3f2edc27c1153500a43b8b0b720ba3ca1f287ded (rerequested code review)

File: vector/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -2526,18 +2526,26 @@
     <string name="ftue_auth_carousel_2_title">You\'re in control.</string>
     <!-- TODO TO BE REMOVED -->
     <string name="ftue_auth_carousel_2_body">Element lets you choose where your messages are stored, keeping you in control of your data.</string>
-    <string name="ftue_auth_carousel_3_title">Secure Messaging.</string>
+    <string name="ftue_auth_carousel_3_title">Secure messaging.</string>
```
Comment (bmarty) [inline vector/src/main/res/values/strings.xml:2529]: I see the change from `M` to `m`. 👍 

FTR On the stores we use "Secure Messenger". [Source](https://github.com/vector-im/element-android/blob/main/fastlane/metadata/android/en-US/title.txt#L1)

image

image

Could be nice to harmonize? CC @amshakal @daniellekirkwood

**Review by bmarty — APPROVED** [review]
LGTM, just added my 2 cents

File: vector/src/main/res/layout/fragment_ftue_auth_splash.xml [hunk 2]
```diff
@@ -182,7 +182,6 @@
         style="@style/Widget.Vector.Button.Login"
         android:layout_width="0dp"
         android:layout_height="wrap_content"
-        android:text="@string/login_splash_submit"
```
[the remark this answers is not in the document]
Comment (ouchadam) [reply vector/src/main/res/layout/fragment_ftue_auth_splash.xml:185]: added https://github.com/vector-im/element-android/pull/4964/commits/3c7f2c5bdfa8b2cf69531f6946ab8b33de7bd9d3

**Pull request merged**